src/main/gui/window.ts

```diff
@@ -19,7 +19,7 @@ export async function createWindow(): Promise<void> {
     );
     // eslint-disable-next-line require-atomic-updates
     win = new BrowserWindow({
-        webPreferences: {nodeIntegration: true},
+        webPreferences: {nodeIntegration: true, webSecurity: false},
```

Comment:
?

Reply:
Enabling CORS requests. Prysm API doesn't support any config for that

Reply:
Can you open issue for that on prysmatic github and issue here(to remove this once they've implemented fix)?